mock/py/mock.py

```diff
@@ -63,568 +62,568 @@
-
-    parser.add_option("-p", "--print-root-path", help="print path to chroot root",
-                      dest="printrootpath", action="store_true",
-                      default=False)
-
-    parser.add_option("-l", "--list-snapshots",
-                      help="list LVM/overlayfs snapshots associated with buildroot",
-                      dest="list_snapshots", action="store_true",
-                      default=False)
-
-    # SCM options
-    parser.add_option("--scm-enable", help="build from SCM repository",
-                      dest="scm", action="store_true",
-                      default=None)
-    parser.add_option("--scm-option", action="append", dest="scm_opts",
-                      default=[], type=str,
-                      help="define an SCM option (may be used more than once)")
-
-    # Package management options
-    parser.add_option("--yum", help="use yum as package manager",
-                      dest="pkg_manager", action="store_const", const="yum")
-    parser.add_option("--dnf", help="use dnf as package manager",
-                      dest="pkg_manager", action="store_const", const="dnf")
-
-    # Bootstrap options
-    parser.add_option('--bootstrap-chroot', dest='bootstrapchroot', action='store_true',
-                      help="build in two stages, using chroot rpm for creating the build chroot",
-                      default=None)
-    parser.add_option('--no-bootstrap-chroot', dest='bootstrapchroot', action='store_false',
-                      help="build in a single stage, using system rpm for creating the build chroot",
-                      default=None)
-
-    parser.add_option('--use-bootstrap-image', dest='usebootstrapimage', action='store_true',
-                      help="create bootstrap chroot from container image (turns "
-                           "--bootstrap-chroot on)", default=None)
-    parser.add_option('--no-bootstrap-image', dest='usebootstrapimage', action='store_false',
-                      help="don't create bootstrap chroot from container image", default=None)
-
-    parser.add_option('--additional-package', action='append', default=[],
-                      type=str, dest="additional_packages",
-                      help=("Additional package to install into the buildroot before "
-                            "the build is done.  Can be specified multiple times."))
-
-    (options, args) = parser.parse_known_args()
-
-    if options.mode == '__default__':
-        # handle old-style commands
-        if len(args) and args[0] in ('chroot', 'shell', 'rebuild', 'install',
-                                     'installdeps', 'remove', 'init', 'clean'):
-            options.mode = args[0]
-            args = args[1:]
-        else:
-            options.mode = 'rebuild'
+    from mockbuild.constants import MOCKCONFDIR, PYTHONDIR, VERSION
+    from mockbuild.file_downloader import FileDownloader
+    from mockbuild.mounts import BindMountPoint, FileSystemMountPoint
```

Comment:
Hmmm this indent change is probably unwanted.

Reply:
What? How this happened?
Fixed and updated.